test/cases/load_test.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/bash
+
+#
+# The image-builder Load test
+#
+# This script sets `-x` and is meant to always be run like that. This is
+# simpler than adding extensive error reporting, which would make this script
+# considerably more complex. Also, the full trace this produces is very useful
+# for the primary audience: developers of image-builder looking at the log
+# from a run on a remote continuous integration system.
```

Comment:
Explaining set -x is maybe a bit much here :D

Reply:
Lol, I just copied the header from the other test cases ^^

Reply:
Oh lol, in that case it's fine I suppose

Reply:
Nah, you were right, I'm removing this comment, it's a bit too much.